lib/tasks_private/vpc.rake

```diff
@@ -0,0 +1,168 @@
+namespace :spec do
+  require 'ibm_cloud_sdk_core'
+  require 'ibm_cloud_resource_controller'
+  require 'ibm_vpc'
+
+  base_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.config.root.to_s, 'spec')
+  cass_dir = File.join(base_dir, 'vcr_cassettes/manageiq/providers/ibm_cloud')
+  spec_dir = File.join(base_dir, 'models/manageiq/providers/ibm_cloud')
+
+  # Prerequisites:
+  # 1. Log in with IBM Cloud CLI i.e. `ibmcloud login -sso`
+  # 2. Place IBM Cloud API key in secrets config
+  desc "Generate refresher VCR"
+  task :refresh_init => :environment do
+    Dir["#{cass_dir}/vpc/cloud_manager/refresher*.yml"].each do |file|
+      FileUtils.rm(file, :verbose => true)
+    end
+    Dir["#{cass_dir}/vpc/cloud_manager/refresher/vm_target.yml"].each do |file|
+      FileUtils.rm(file, :verbose => true)
+    end
+
+    apikey = YAML.load_file("config/secrets.yml")["test"]["ibm_cloud_vpc"]["api_key"]
+    token = IBMCloudSdkCore::IamAuthenticator.new(:apikey => apikey)
+                                             .instance_variable_get(:@token_manager).access_token
+    auth = IBMCloudSdkCore::BearerTokenAuthenticator.new(:bearer_token => token)
+
+    connection = IbmVpc::VpcV1.new({:authenticator => auth})
+    resource_controller = IbmCloudResourceController::ResourceControllerV2.new({:authenticator => auth})
+
+    # Create resources
+    begin
+      default_resource_group_id = `ibmcloud resource group default --id | tr -d '\n'`
+      cloud_db_id = resource_controller.create_resource_instance(
+        :name             => 'rake-db',
+        :target           => 'bluemix-us-south',
+        :resource_group   => default_resource_group_id,
+        :resource_plan_id => 'databases-for-postgresql-standard'
+      ).result['id']
+      status = resource_controller.get_resource_instance(:id => cloud_db_id).result['state'] until status == 'active'
+    rescue => err
+      puts err
```

Comment:
So if this create fails this will continue on, is that intentional (aka ignore any failures here)?

Reply:
I've decided to `raise` here instead since if we were to continue https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb#L29 would fail anyway